rom-elks.c

```diff
@@ -393,25 +348,6 @@ static int int_13h ()
 	}
 
 
-// BIOS misc services
-
-static int int_15h ()
-	{
-	byte_t ah = reg8_get (REG_AH);
-	switch (ah)
-		{
-		// Return CF=1 for all non implemented functions
-		// as recommended by Alan Cox on the ELKS mailing list
-
-		default:
-			flag_set (FLAG_CF, 1);
-
-		}
-
-	return 0;
-	}
-
-
```

Comment:
https://github.com/jbruchon/elks/commit/4ef5e84e2d8ea3fcdd213da9e0c5a92c30712be5 ?

Reply:
Thanks for pointing that out. We could leave it in, but as you can see in the original commit, that code was never updated from BCC to gcc compiler, so INT 15h will never be called. IMO, ELKS APM is completely inoperational at this time. I figure we should remove it until it is actually compilable by ELKS. (It is saved in rom-pcxtat.c of course).

Reply:
Okay...